### DISCUSSION TITLE
Ability to combine regions for gen 2 (or genlockes etc)

### DISCUSSION THREAD

**Discussion by JonnyMillar29** [jynnie/soullocke#19] [Ideas]
Hi hi!
I'm currently in the middle of a Soul Silver run, and would like to be able to track all of our encounters through the whole game in one place - we've currently got 2 pages going, one for Johto and then one for Kanto, but switching back and forth between them is a pain. And if for example something we picked up in Johto dies on a route in Kanto, there's no way to track that at the moment.

Even if you're just doing a gen 2 run to the first run through of the e4, that still requires you to go into Kanto and there's a couple routes and victory road there that aren't in Johto.

Maybe a game selector as opposed to / as well as a region selector when setting up would work?
Or just an option to add a region to your current tracker.
Maybe you're playing a genlocke and want to track your run through multiple generations and regions.

Huge fan of the tracker, and really appreciate the work you put in :)

Thanks!

**Comment by jynnie** [comment]
Gotcha, I'll look into it! In the meantime, you can put in custom name routes.
![img](https://user-images.githubusercontent.com/7284723/221387022-e2da224d-1db4-4fa0-86ee-095bbaeea460.gif)

**Reply by JonnyMillar29** [reply]
Great, thanks!
